service/docs/source/geopm_pio.7.rst

```diff
@@ -73,11 +73,20 @@ Descriptions Of High Level Aliases
 
 ``BOARD_ENERGY``
     Total energy measured on the server's board. See :ref:`geopm_pio_cnl(7)
-    <geopm_pio_cnl.7:Requirements>` for signal availability requirements.
+    <geopm_pio_cnl.7:Requirements>` and :doc:`geopm_pio_msr(7)
+    <geopm_pio_msr.7>` for signal availability requirements.
 
 ``BOARD_POWER``
     Power measured on the server's board. See :ref:`geopm_pio_cnl(7)
-    <geopm_pio_cnl.7:Requirements>` for signal availability requirements.
+    <geopm_pio_cnl.7:Requirements>` and :doc:`geopm_pio_msr(7)
+    <geopm_pio_msr.7>` for signal availability requirements.
```

Comment:
Any of these aliased signals that collide between CNL and MSR should have a similar blurb about preference.